net/relayd/src/opnsense/mvc/app/models/OPNsense/Relayd/Relayd.php

```diff
@@ -38,4 +38,19 @@
  */
 class Relayd extends BaseModel
 {
+    /**
+     * get, set, unset configuration state
+     * @param $action - null = get, true = set, false = unset
+     * @return bool
+     */
+    public function configChanged($status = null)
```

Comment:
The function mixes purposes, incomparable values. phpdoc says $action, but it uses $status here

Reply:
Best to split this into configChanged(), configDirty(), configClean()

Reply:
Huch, wasn't aware that bool can't be null.
status vs. action is my fault.